packages/base-types/src/trace/index.ts

```diff
@@ -15,6 +15,7 @@ export { TraceFrame as SpeakTrace } from '@/node/speak';
 export { TraceFrame as StreamTrace } from '@/node/stream';
 export { TraceFrame as TextTrace } from '@/node/text';
 export { TraceFrame as VisualTrace } from '@/node/visual';
+export { TraceType } from '@/node/trace';
```

Comment:
looks like `TraceType` is defined in the node utils

```suggestion
export { TraceType } from '@/node/utils/trace';
```